docs/resources/grant_privileges_to_database_role.md

```diff
@@ -6,6 +6,8 @@ description: |-
   
 ---
 
+!> **Warning** This is a preview resource. It's not production-ready (yet), and some of the edge cases may not be covered. In case of any errors, please file an issue in our GitHub repository.
```

Comment:
Why these two warnings are different (this one and the one from account role)? This one seems a bit too hard. I would like people to use it on production to give feedback. I think we can weaken this one.

I am also wandering if these should be warnings or notes (yellow vs blue text boxes). WDYT?